en/index.md

```diff
@@ -53,6 +53,61 @@ technology terms:
  - space-separated, comma-delimited
  - load balancer (only upper case as part of proper name e.g. Elastic Load Balancer)
 
+## Branding
+
+Consistency in branding is important for a number of reasons, including the protection of trademarks where they apply. 
```

Comment:
Should we include what is required for consistency in branding? Even something as simple as a redirect URL to branding resources or a guidance could help. Otherwise, I am afraid, this statement does not provide an actionable path.

Reply:
What do you expect that to contain? The text following outlines the proper way of identifying Ubuntu and product names. What else would we add?

Reply:
Never mind, I misread it (was not in my usual view and I didn't see this was the opening sentence to the following guidelines). Sorry.